### PR TITLE
Feature/122 cleanup tabs styles

### LINKED ISSUE #122: Improve readability & intuitiveness of the Sidebar link titles
* FAQ - might be better as Frequently Asked Questions
* Log - the disabled style is pretty subtle and it might not be obvious what that button is for until after the model starts running. We could gray it out much more, and maybe expand the name to "View Log" or "View Model Log"
* User's Guide & FAQ - both could benefit from an icon that indicates this link opens in a new browser ( we could leave this out until after #102 to avoid changing it multiple times)

If it's useful to add further explanation to any of these items, we could do so in some hover text.

### PR DESCRIPTION
**Pull request by davemfish**
Various style & layout improvements here, mainly focused on keeping things clean & tidy at varying window widths. Fewer items are allowed to wrap, the overall window has a strict minimum 800 pixel width. Most of the attention went to items in the top navbar, and then the two main columns on each of the tabs.

This Fixes #122 and addresses a couple items in #87

### CODE REVIEW

**Comment by davemfish** [comment]
Here's a before and after screenshots to show some new styling of the tabs. (These are from puppeteer in GHA and that's why they don't display the window frame properly)

Before:
invest-workbench-4-complete-setup-form

After:
invest-workbench-4-complete-setup-form

And another after showing how multiple tabs now shrink as needed instead of having a horizontal scroll bar:
Capture

**Comment by davemfish** [comment]
> After:

Just noticed the feedback font size got bigger here, that was inadvertent and I just pushed a change to make it smaller again.

File: tests/renderer/app.test.js [hunk 1]
```diff
@@ -205,7 +205,9 @@ describe('Various ways to open and close InVEST models', () => {
     expect(tab3EventKey).not.toEqual(tab1EventKey);
 
     // Click the close button on the middle tab
-    const tab2CloseButton = await within(tab2).getByRole('button', { name: /x/ });
+    const tab2CloseButton = await within(tab2.closest('.nav-item'))
+      .getByRole('button', { name: /x/ });
+    // const tab2CloseButton = await within(tab2).getByRole('button', { name: /x/ });
```
Comment (dcdenu4) [inline tests/renderer/app.test.js:210]: Good to remove this commented line?

**Review by dcdenu4 — CHANGES_REQUESTED** [review]
Hey @davemfish this looks really good. I tested out your branch locally and everything resized very nicely. The only thing I noticed was that the "LOG" wording hasn't been changed. I think in one of the linked issues, it mentioned this being a little confusing and that maybe "View Log" or something might make more sense. Thoughts?

**Comment by dcdenu4** [comment]
>Just noticed the feedback font size got bigger here, that was inadvertent and I just pushed a change to make it smaller again.

Great! I did notice the font was bigger when running a model, but I didn't mind it haha. Hopefully that doesn't mean I'm getting old.

File: tests/renderer/app.test.js (hunk 1 above)
Comment (davemfish) [reply tests/renderer/app.test.js:210]: Thank you!

**Comment by davemfish** [comment]
> > Just noticed the feedback font size got bigger here, that was inadvertent and I just pushed a change to make it smaller again.
> 
> Great! I did notice the font was bigger when running a model, but I didn't mind it haha. Hopefully that doesn't mean I'm getting old.

Yeah this is something I still don't fully understand. We're using `rem` units for fonts everywhere, and I think the baseline font size must have changed for the text in the setup form, but I don't know exactly why, maybe as a side-effect of some of the other changes there. Anyway I agree it's suitable as-is and can easily be adjusted more.

**Comment by davemfish** [comment]
> Hey @davemfish this looks really good. I tested out your branch locally and everything resized very nicely. The only thing I noticed was that the "LOG" wording hasn't been changed. I think in one of the linked issues, it mentioned this being a little confusing and that maybe "View Log" or something might make more sense. Thoughts?

Thanks for trying it out! I thought "Log" was a pretty good label after all. It's one of those things where the very first time you see it you might not know what it's about. But it's also disabled at that point and it enables and switches to it automatically when the user clicks "Run". And from that point on, I think the user will know what "Log" means.

**Comment by davemfish** [comment]
Hmm we're getting annoying npm errors on the `npm install` step our workflows, before it even tries installing anything. It's been an intermittent problem. I just bumped the node version in the workflow files in case that helps.

**Comment by dcdenu4** [comment]
hmmm now I'm seeing a mac failure from this test:

https://github.com/natcap/invest-workbench/runs/3001910418?check_suite_focus=true

**Comment by davemfish** [comment]
> hmmm now I'm seeing a mac failure from this test:
> 
> https://github.com/natcap/invest-workbench/runs/3001910418?check_suite_focus=true

Yeah I just made #155 for that. I don't think it's related to these changes.

**Review by dcdenu4 — APPROVED** [review]
Approved.

**Comment by dcdenu4** [comment]
>Hmm we're getting annoying npm errors on the npm install step our workflows, before it even tries installing anything. It's been an intermittent problem. I just bumped the node version in the workflow files in case that helps.

Judging by the tests this either helped or didn't show itself. So I'll go ahead and merge as it's not related to this PR.

**Pull request merged**